sources/api/controllers/UserController.ts

```diff
@@ -208,27 +215,25 @@ export async function deleteUser(req: Request, res: Response) {
 // Get all groups a User belongs to
 //
 export async function getUserGroups(req: Request, res: Response) {
-    const conn = await DbConnection.getConnection();
-    const userRepository = conn.getRepository(User);
-    const user = await userRepository.findOne({where: {pseudo: req.params.pseudo}});
-    const requesting_user = getRequestingUser(req);
-
-    if (!user) {
-        throw new ApiException(404, "User not found");
-    }
-    if (user.id !== requesting_user.id) {
-        throw new ApiException(403, "Action not allowed")
+    let user: User = getRequestingUser(req);
```

Comment:
This is the first time that you add the `: User` for `getRequestingUser`, I find it redundant, but if you wish to add it you will need to add it in the other calls :heart:

Reply:
In fact, the type should not be specified here